src/client/scripts/views/SignedIn/DiscoverView.js

```diff
@@ -157,10 +165,89 @@ export class DiscoverView {
      */
     async #renderFilterBar() { // TODO: Gauri
         const elm = document.createElement('div');
+        // elm.style.marginBottom = "100px";
 
         // TODO: Render the filter bar - should have checkboxes associated with the character property in the User data structure,
         // and just a few properties in the Preferences data structure (otherwise it's too much work). Only add the Preferences
         // properties if !this.#curUser.hasHousing
+        
+        // Dictionary for filtering users 
+        const filterOptions = {
+            "Gender": {
+                "female": false,
+                "male": false,
+                "nonbinary": false
+            },
+            "Clean": {
+                "messy": false,
+                "clean": false,
+                "very clean": false
+            },
+            "Sleep": {
+                "morning riser": false,
+                "afternoon riser": false,
+                "evening riser": false
+            },
+            "Noise": {
+                "low noise": false,
+                "medium noise": false,
+                "loud noise": false
+            },
+            "Guests": {
+                "no to less guests": false,
+                "some guests": false,
+                "any number of guests": false
```

Comment:
Could you change the slider values to their property names in User (so all lowercase) and the keys in the inner object to 1-3? it'll make it easier to implement.
```
'clean': {  '1': false, '2': false, '3': false }
```